src/hooks/useFormState.js

```diff
@@ -70,7 +70,7 @@ export function useFormState(initialState) {
   // Verify form has been iteractedWith and also has no errors;
   // Used for determining whether a form is valid
   function getFormIsValid() {
-    return touched && !getFieldErrorsString()
+    return iteractedWith && !getFieldErrorsString()
```

Comment:
This is the bugfix for https://app.asana.com/0/1136962714401929/1145702600849513/f